src/components/Filter/Sidebar.jsx

```diff
@@ -0,0 +1,187 @@
+              checked={libas}
+            />
+            <h3 className="content-3a">LIBAS</h3>
+          </div>
+          <div className="checkbox-content">
+            <input
+              type="checkbox"
+              value="varanga"
+              name="brand"
+              className="checkbox-type"
+              onChange={() => filterDispatch({ type: "VARANGA" })}
+              checked={varanga}
+            />
+            <h3 className="content-3a">VARANGA</h3>
+          </div>
+        </div>
+      </div>
+
+      <div className="sidebar-content-2">
+        <h3 className="content-1a">Discount</h3>
+        <div className="sidebar-content-4a">
+          <div className="radio-content">
+            <input
+              type="radio"
+              value="80"
+              name="price-discount"
+              onChange={(e) =>
+                filterDispatch({
+                  type: "SORT_BY_DISCOUNT",
+                  value: e.target.value
+                })
+              }
+              checked={discount === 80}
+            />
+            <h3 className="content-3a">80% & above</h3>
+          </div>
+          <div className="radio-content">
+            <input
+              type="radio"
+              value="60"
+              name="price-discount"
+              onChange={(e) =>
+                filterDispatch({
+                  type: "SORT_BY_DISCOUNT",
+                  value: e.target.value
+                })
+              }
+              checked={discount === 60}
+            />
+            <h3 className="content-3a">60% & above</h3>
+          </div>
+          <div className="radio-content">
+            <input
+              type="radio"
+              value="40"
+              name="price-discount"
+              onChange={(e) =>
+                filterDispatch({
+                  type: "SORT_BY_DISCOUNT",
+                  value: e.target.value
+                })
+              }
+              checked={discount === 40}
+            />
+            <h3 className="content-3a">40% & above</h3>
+          </div>
+          <div className="radio-content">
+            <input
+              type="radio"
+              value="20"
+              name="price-discount"
+              onChange={(e) =>
+                filterDispatch({
+                  type: "SORT_BY_DISCOUNT",
+                  value: e.target.value
+                })
+              }
+              checked={discount === 20}
+            />
+            <h3 className="content-3a">20% & above</h3>
+          </div>
+        </div>
+      </div>
+
+      <div className="sidebar-content-2">
+        <h3 className="content-1a">Sort by</h3>
+        <div className="sidebar-content-4a">
+          <div className="radio-content">
+            <input
+              type="radio"
+              value="low"
+              name="group"
+              onChange={() =>
+                filterDispatch({ type: "SORT_BY_PRICE", payload: "lowToHigh" })
```

Comment:
You could add type: "SORT_BY_PRICE" in a constant to avoid typo error and for ease of reuse.